utils/build-script

```diff
@@ -445,18 +445,12 @@ class BuildScriptInvocation(object):
             self.platforms_to_skip_test.add(StdlibDeploymentTarget.Cygwin)
         if args.skip_test_osx:
             self.platforms_to_skip_test.add(StdlibDeploymentTarget.OSX)
-        if args.skip_test_ios_host:
-            self.platforms_to_skip_test.add(StdlibDeploymentTarget.iOS)
```

Comment:
Aren't we *supposed* to disable these based on `skip_test_ios`, though? ... etc for lines below ...